### DISCUSSION TITLE
請問有任何方法可以在C++中運行CnOCR嗎？

### DISCUSSION THREAD

**Discussion by DingHsun** [breezedeus/CnOCR#288] [General]
我想在C++的專案中運行CnOCR文本判別，請問有釋出C++版本的CnOCR嗎？或者說我只能將py轉成exe檔並在C++中執行呢？

**Comment by breezedeus** [comment]
https://github.com/CoireLen/CnocrCpp 试试这个

**Reply by DingHsun** [reply]
謝謝您!用此方法得到解決!
